File: VetBooking/Models/Meeting.cs
using System;
using VetBooking.Areas.Identity.Data;

namespace VetBooking.Models
{
    public class Meeting
    {
        public int ID { get; set; }

        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Date { get; set; }

       public VetBookingUser User { get; set; }
    }
}

File: app/Models/Meeting.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Meeting extends Model
{
    public $timestamps = false;

    protected $fillable = [
        'name',
        'description',
        'date',
        'firstname',
        'lastname',
        'phone_number',
        'petname',
        'state'
    ];

    protected $casts = [
        'date' => 'datetime',
    ];
}
